### PR TITLE
Documetation for new 'fsync' imjournal option

### PR DESCRIPTION
**Pull request by jvymazal**
related to #3762 main repo PR

### CODE REVIEW

**Comment by jvymazal** [comment]
link to main code PR: https://github.com/rsyslog/rsyslog/pull/3762

File: source/configuration/modules/imjournal.rst [hunk 1]
```diff
@@ -245,7 +245,7 @@ WorkAroundJournalBug
    :widths: auto
    :class: parameter-table
 
-   "binary", "off", "no", "none"
```
Comment (rgerhards) [inline source/configuration/modules/imjournal.rst]: is this a doc fix or a change-of-behavior?

Comment (jvymazal) [reply source/configuration/modules/imjournal.rst]: Doc fix - the actual default was changed in https://github.com/rsyslog/rsyslog/pull/3566

Comment (rgerhards) [reply source/configuration/modules/imjournal.rst]: thx!

**Pull request merged**